.palantir/revapi.yml

```diff
@@ -0,0 +1,2 @@
+versionOverrides:
+  com.squareup.misk:misk-action-scopes:misk-0.18.0: "2023.10.02.170330-4694630"
```

Comment:
Revapi uses git tags to find the most recent version to compare against. Looks like we stopped adding git tags a long time ago. This is my hacky solution to work around that for this demo.

To use revapi for real, we'd need to either start adding git tags on release, or write version overrides to this files on release (there is gradle task provided by revapi for this). Neither sound like blocking issues?